spaces/S000126/README.md

```diff
@@ -1,17 +1,17 @@
 ---
 uid: S000126
-slug: cantor's-teepee
-name: Cantor's Teepee
+slug: punctured-kn-fan
+name: Punctured Knaster-Kuratowski fan
 aliases:
-  - Punctured Knaster-Kuratowski fan
+  - Cantor's Teepee
 counterexamples_id: 129
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
 - wikipedia: Knaster-Kuratowski_fan
   name: Knaster-Kuratowski fan
 ---
-Cantor's Teepee is Cantor's Leaky Tent with the apex removed. Specifically: For any $a \in [0,1]$, let $L(a)$ be the line segment from $(a,0)$ to $p = (\frac{1}{2}, \frac{1}{2})$. Let $\mathcal{C}$ be the middle-thirds Cantor set in the unit interval, $\mathcal{E}$ the endpoints of the removed intervals and $\mathcal{F} = \mathcal{C} \setminus \mathcal{E}$. Define $A = \{(x,y) \in L(c)\ |\ c \in \mathcal{E}, y \in \mathbb{Q}\}$ and $B = \{(x,y) \in L(c)\ |\ c \in \mathcal{F}, y \not\in \mathbb{Q}\}$. Cantor's Teepee is $X = A \cup B \setminus\{p\} \subset \mathbb{R}^2$ with the subspace topology.
+{S000125} with the apex removed. Specifically: For any $a \in [0,1]$, let $L(a)$ be the line segment from $(a,0)$ to $p = (\frac{1}{2}, \frac{1}{2})$. Let $\mathcal{C}$ be the middle-thirds Cantor set in the unit interval, $\mathcal{E}$ the endpoints of the removed intervals and $\mathcal{F} = \mathcal{C} \setminus \mathcal{E}$. Define $A = \{(x,y) \in L(c)\ |\ c \in \mathcal{E}, y \in \mathbb{Q}\}$ and $B = \{(x,y) \in L(c)\ |\ c \in \mathcal{F}, y \not\in \mathbb{Q}\}$. This space is $X = A \cup B \setminus\{p\} \subset \mathbb{R}^2$ with the subspace topology.
```

Comment:
One minor thing: should the last line use $X = (A \cup B) \setminus\\{p\\} \subset \mathbb{R}^2$ to avoid ambiguity?  (I can't take the time to parse if $p$ belongs to $A$ or not)